go.mod

```diff
@@ -1,35 +1,23 @@
 module github.com/trisacrypto/directory
 
-go 1.16
+go 1.17
```

Comment:
@rebeccabilbro ok, so now this PR is officially out of hand - looks like I need Go 1.17 to finish this task since I'm relying on something in the `http` standard library. Should we just push through and go with this or should we discuss as a group?